server_list/templates/address_verification_failed.txt

```diff
@@ -0,0 +1,10 @@
+Server address does not include a DNS record for the IP that the announcement was sent from.
+Announce IP: {{ announce_ip }}
+Address records: {{ valid_addresses | join ", " }}
+
+Help: This is usually because your server is only listening on IPv4 but your announcement is being sent over IPv6.
+There are two ways to fix this:
+ 1. (preferred) Set bind_address = :: in your server config to listen on IPv6 and add your IPv6 address to DNS as an AAAA record.
+    This allows both IPv4 and IPv6 because the linux kernel will forward IPv4 requests to the IPv6 socket by default using an IPv4-mapped IPv6 address (look up IPV6_V6ONLY for more information).
+    On other operating systems this option may not work and you'll have to use the second option.
```

Comment:
It works on every relevant OS and I don't think we need this explanation anyway.

Reply:
Does Windows do IPv6 mapping by default?  That's what I was mainly concerned about.

Reply:
It doesn't but Minetest sets this socket option.